### PR TITLE
Enable component test in Jest L36 to L65 - Closes #1433 #1434 #1435

### LINKED ISSUE #1433: Enable component tests in Jest L36-L45
Fix should be fixed so that these lines can be uncommented and Jest tests pass 

https://github.com/LiskHQ/lisk-hub/blob/ddaa5709fafd6f91bcca20a90a770dc4335c5b3c/jest.config.js#L36-L45

### PR DESCRIPTION
**Pull request by osvaldovega**
### What issue have I solved?

-- #1433 
-- #1434 
-- #1435 

### How have I implemented/fixed it?
In the **jest.config.js** file enable from line 36 to line 65 the components to test, then run the command **npm run test-jest** and check the error messages and fix one by one to confirm that all test pass.

### How has this been tested?
Enabling the components to test in the jest config file, running jest and fix the errors that showed in the terminal one by one

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: src/components/authenticate/authenticate.test.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ describe('Authenticate', () => {
   });
 
   it('should render InfoParagraph with appropriate message', () => {
-    expect(wrapper.find('InfoParagraph').text()).to.include(`You are looking into a saved account. In order to ${props.nextAction} you need to enter your passphrase`);
+    expect(wrapper.find('InfoParagraph').text(`You are looking into a saved account. In order to ${props.nextAction} you need to enter your passphrase`));
```
Comment (slaweet) [inline src/components/authenticate/authenticate.test.js]: The assertion should be 
`expect(wrapper.find('InfoParagraph')).to.have.text(...` 
the current test won't fail if the text is changed.

File: config/setupJest.js [hunk 2]
```diff
@@ -8,6 +8,8 @@ import sinonStubPromise from 'sinon-stub-promise';
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import i18next from 'i18next';
+// TODO remove next line after upgrading node version to at least 7
+import 'babel-polyfill';
```
Comment (slaweet) [inline config/setupJest.js]: Jenkins failed with `JavaScript heap out of memory`

https://jenkins.lisk.io/job/lisk-hub/job/PR-1475/1/consoleFull

File: src/components/authenticate/authenticate.test.js (hunk 1 above)
Comment (osvaldovega) [reply src/components/authenticate/authenticate.test.js]: change done

File: config/setupJest.js (hunk 2 above)
Comment (osvaldovega) [reply config/setupJest.js]: change done

**Pull request merged**